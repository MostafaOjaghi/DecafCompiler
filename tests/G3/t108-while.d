void main () {
	int a;
	int b;
	a = 0;
	b = 9;
	while (a < b) {
		Print(a,b);
		a = a + 1;
		b = b - 1;
	}
}