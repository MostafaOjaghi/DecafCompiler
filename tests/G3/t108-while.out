09
18
27
36
45
